rapthor/lib/miscellaneous.py

```diff
@@ -435,6 +435,28 @@ def get_reference_station(soltab, max_ind=None):
     return ref_ind
 
 
+def remove_soltabs(solset, soltabnames):
+    """
+    Remove H5parm soltabs from a solset
+
+    Note: the H5pram must be opened with readonly = False
+
+    Parameters
+    ----------
+    solset : losoto solution set object
+        The solution set from which to remove soltabs
+    soltabnames : list
+        Names of soltabs to remove
+    """
+    soltabnames = string2list(soltabnames)
+    for soltabname in soltabnames:
+        try:
+            soltab = solset.getSoltab(soltabname)
+            soltab.delete()
+        except Exception:
```

Comment:
Is it possible to catch a less generic exception type?

Reply:
I checked the exception raised by losoto, and indeed it is just Exception.